tests/test_modeling_common.py

```diff
@@ -3846,6 +3855,37 @@ def get_mean_reldiff(failcase, x, ref, atol, rtol):
                                     if "attention_mask" in inspect.signature(model_eager.forward).parameters:
                                         processed_inputs["attention_mask"] = dummy_attention_mask
 
+                                if (
+                                    "bool_masked_pos" in inspect.signature(model_eager.forward).parameters
+                                ) and not deactivate_mask:
+                                    dummy_mask = torch.ones((self.model_tester.num_masks,))
+
+                                    # In case of additional token (like class) we define a custome `mask_length`
+                                    if hasattr(self.model_tester, "mask_length"):
+                                        dummy_mask = torch.cat(
+                                            [
+                                                dummy_mask,
+                                                torch.zeros(self.model_tester.mask_length - dummy_mask.size(0)),
+                                            ]
+                                        )
+                                    else:
+                                        dummy_mask = torch.cat(
+                                            [
+                                                dummy_mask,
+                                                torch.zeros(self.model_tester.seq_length - dummy_mask.size(0)),
+                                            ]
+                                        )
```

Comment:
nit - we can simplify this a bit. This is really a matter of opinion - feel free to add or not 

```suggestion
                                    # In case of additional token (like class) we define a custom `mask_length`
                                    if hasattr(self.model_tester, "mask_length"):
                                        mask_length = self.model_tester.mask_length - dummy_mask.size(0)
                                    else:
                                        mask_length = torch.zeros(self.model_tester.seq_length - dummy_mask.size(0))

                                    dummy_mask = torch.cat([dummy_mask, torch.zeros(mask_length)])
```